nidaqmx/system/system.py

```diff
@@ -34,13 +37,24 @@ class System(object):
     operations on DAQ hardware, and creates classes from which you can get
     information about the hardware.
     """
+    def __init__(self, interpreter = None):
+        self._interpreter = interpreter
 
     @staticmethod
     def local():
         """
         nidaqmx.system.system.System: Represents the local DAQmx system.
         """
-        return System()
+        interpreter = _library_interpreter.LibraryInterpreter(encoding='windows-1251')
+        return System(interpreter=interpreter)
+    
+    @staticmethod
+    def remote(grpc_options):
```

Comment:
Would it make sense for grpc_options to be keyword-only here? I guess not, because it's currently not optional.

Reply:
for system when the user is using the remote method, then the gRPC options becomes a required parameter right?

Reply:
That's what makes sense today because remote() only supports gRPC.

If remote() supported other transports, such as HTTP or Thrift, then the answer would be different.